src/components/BookList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from './Book';
+
+const mapStateToProp = state => ({ books: state.bookReducer });
+
+const BookList = ({ books }) => {
+  const list = [];
+  books.forEach(book => {
+    list.push(
+      <Book key={book.ID} book={book} />,
+    );
+  });
+  return (
```

Comment:
- [ ] Rather than a forEach simple map through the books array and display it. It's more concise.

Reply:
Done.